src/app/app.component.scss

```
@@ -0,0 +1,33 @@
+.formControl {
```

для классов используй kebab-case
```suggestion
.form-сontrol {
```